### PR TITLE
docs(reference): Add inclusive gateway to bpmn symbol support matrix.

### LINKED ISSUE #1126: I can read about Inclusive Gateway
The diverging inclusive gateway is added to Zeebe in https://github.com/camunda/zeebe/pull/9747 as an external contribution. Of course, all the Camunda Platform 8 need to be aligned and support the feature.

We need to document how users can use the diverging Inclusive Gateway and how it behaves. The behavior is specified in the PR.

Special modeling rules exists, and can be found in:
- https://github.com/camunda/camunda-modeler/issues/3083

Link to product hub:
- https://github.com/camunda/product-hub/issues/214

### PR DESCRIPTION
**Pull request by skayliu**
## What is the purpose of the change

Document how users can use the diverging Inclusive Gateway and how it behaves, The behavior is specified in the PR [https://github.com/camunda/zeebe/pull/9747](https://github.com/camunda/zeebe/pull/9747).

closes #1126

## Are there related marketing activities

Not that we're aware.

## When should this change go live?

_Does this change need to be released before a certain date or milestone? If so, when?_

**Should be released in 8.1**

## PR Checklist

- [ ] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by skayliu** [comment]
Hi,@korthout ,@menski i've docs for [I can read about Inclusive Gateway](https://github.com/camunda/camunda-platform-docs/issues/1126#top)，please check the PR，thank you .

File: docs/components/modeler/bpmn/inclusive-gateways/inclusive-gateways.md
Comment (korthout) [inline docs/components/modeler/bpmn/inclusive-gateways/inclusive-gateways.md:18]: ❌ All images must have descriptive alt-texts according to the [styleguide](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/technical-writing-styleguide.md#formatting-organization-and-structure-for-implementation-steps). Please replace `process` with a full description of what the image shows. This improves the accessibility of the docs.

❌ These images were useful for the PR, but don't provide the same level of information as the images on other gateway pages (i.e. abstract task names, no labels on the gateway, and sequence flows. It would be best to follow our own best practices on [labeling elements](https://docs.camunda.io/docs/next/components/best-practices/modeling/creating-readable-process-models/#labeling-bpmn-elements). Please replace the images, so they:
 - are labeled according to our [best practices](https://docs.camunda.io/docs/next/components/best-practices/modeling/creating-readable-process-models/#labeling-bpmn-elements)
 - are sized according to our [styleguide](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/technical-writing-styleguide.md#formatting-organization-and-structure-for-implementation-steps) (i.e. All images must be appropriately sized). I'm unsure what the exact limitations are, but the images are very large compared to the rest of the text right now. When resizing the images, please take into account

**Review by korthout — CHANGES_REQUESTED** [review]
Wow, thanks for these docs @skayliu 🚀 

👍 I really like that you've described the entire concept. All relevant technical details are described

❌ Before we can merge this, we need to make a few changes still:
- We need to correct a technical statement (the inclusive gateway can be used to make multiple decisions, not just one)
- We need to align the images with our own best practices
- We need to make sure our technical writing style guide is applied in these docs
- We need to resolve some grammar mistakes. Perhaps @christinaausley can have a look at these

Thanks again @skayliu. As always, please let me know if anything is unclear. 🙇

**Comment by akeller** [comment]
👋 I've added myself as a reviewer in case @christinaausley is still on vacation when this is ready for a style guide/grammar review.

File: docs/components/modeler/bpmn/inclusive-gateways/inclusive-gateways.md
Comment (korthout) [inline docs/components/modeler/bpmn/inclusive-gateways/inclusive-gateways.md:65]: ❌ Please fix the typo in `defulat` -> `default`

File: docs/components/modeler/bpmn/inclusive-gateways/inclusive-gateways.md
Comment (korthout) [inline docs/components/modeler/bpmn/inclusive-gateways/inclusive-gateways.md:77]: ❌ Please fix the typo in `conatins` -> `contains` and reformat `(courses ,"pasta")` -> `(courses, "pasta")`

File: docs/components/modeler/bpmn/inclusive-gateways/inclusive-gateways.md
Comment (korthout) [inline docs/components/modeler/bpmn/inclusive-gateways/inclusive-gateways.md:33]: 🔧 IMO the behavior is already clear with the other examples. I think we can remove this one, but it's fine to keep it. I'll leave it up to you.

File: docs/components/modeler/bpmn/inclusive-gateways/inclusive-gateways.md
Comment (korthout) [inline docs/components/modeler/bpmn/inclusive-gateways/inclusive-gateways.md:58]: ☝️ I just noticed the same typo here as well

**Review by korthout — CHANGES_REQUESTED** [review]
Thanks for applying the requested changes so quickly! 🚀  Well done 👏 

👍 I really like the new examples

❌ Please have a look at my comments. There are still a few things to improve, but we're almost there. 

As always, let me know if you need a hand or have any questions. 🙇

**Review by korthout — APPROVED** [review]
Nice work @skayliu 🚀 LGTM

As this is an external contribution, our team can't add commits on top for the technical writing review. So, we'll merge it and then make some final adjustments in another PR. //cc @akeller 

Thanks again so much for this documentation @skayliu ❤️ We really appreciate it

**Comment by christinaausley** [comment]
@korthout Sorry I missed this! Mind if I launch a PR for technical review, or have you already launched a space for changes?

**Comment by akeller** [comment]
@christinaausley I reviewed a PR this morning, but feel free to take a closer look at https://github.com/camunda/camunda-platform-docs/pull/1247#issuecomment-1242111346

**Comment by korthout** [comment]
@christinaausley I agree with @akeller, please have a look at all the content for the inclusive gateway. For completeness, the other PR is this one:
- https://github.com/camunda/camunda-platform-docs/pull/1247

**Pull request merged**
